de-lib/src/main/java/org/iplantc/de/theme/base/client/tools/NewToolRequestFormViewDefaultAppearance.java

```diff
@@ -114,9 +114,9 @@ public String fileNameValidationMsg() {
 
     @Override
     public String getFileName(String filename) {
-        if (Strings.isNullOrEmpty(filename) || !GXT.isChrome()) {
+        if (Strings.isNullOrEmpty(filename) || !filename.contains(BROWSER_FAKEPATH)) {
```

Comment:
Since we're always returning `filename.substring(12)` if this check fails, maybe should check for `filename.startsWith(BROWSER_FAKEPATH)` instead of `contains`?

Reply:
Good point 👍 . I will make that change.